cfgov/unprocessed/apps/teachers-digital-platform/js/encode-name.js

```diff
@@ -48,54 +44,61 @@ const x = 'x';
 const y = 'y';
 const btoa = xtoy.replace( x, b ).replace( y, a );
 const atob = xtoy.replace( x, a ).replace( y, b );
+const base64Encode = wwindow[btoa];
+const base64Decode = wwindow[atob];
```

Comment:
Why `wwindow` instead of `window`? Is it to avoid two lookups on `window`?

Reply:
The overall original goal was to obfuscate the _minified_ source, but I'm going to clean this up if I can today.